### PR TITLE
Colin's prototype

### PR DESCRIPTION
**Pull request by semicolin**
Copied the code from my prototype

### CODE REVIEW

File: projects/showcase/src/app/datasets/example.service.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          type: 'time',
+          time: {
+            unit: 'day',
+            displayFormats: {
+              day: 'd MMM yyyy',
+            },
+          },
+        },
+      },
+    }).pipe(delay(2000));
```
Comment (yashpmohod) [inline projects/showcase/src/app/datasets/example.service.ts:50]: why do we need a delay here?

Comment (semicolin) [reply projects/showcase/src/app/datasets/example.service.ts:50]: This is example service is just for testing. I was trying to simulate the delay that you would get when making a network request in a real service.

File: projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+  // todo: roll this into stateSubject?
+  private timelineRangeSubject = new ReplaySubject<{ min: number; max: number }>();
+  timelineRange$ = this.timelineRangeSubject.asObservable();
+
+  loading = false;
+
+  retrieveAll() {
+    this.loading = true;
+    merge(...this.dataLayerServices.map((service) => service.retrieve())).subscribe({
```
Comment (yashpmohod) [inline projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.ts:58]: @semicolin, can you please add comment around this, how it exactly performs

Comment (semicolin) [reply projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.ts:68]: added a comment

**Review by yashpmohod — APPROVED** [review]
as such, there is no comment from my side,  going to approve, if I want I will ask to clin

**Pull request merged**